InnerEye/ML/reports/classification_report.py

```diff
@@ -49,15 +49,17 @@ class ReportedMetrics(Enum):
     FalseNegativeRate = "false_negative_rate"
 
 
-def get_results(csv: Path) -> LabelsAndPredictions:
+def get_results(csv: Path, hue: str) -> LabelsAndPredictions:
```

Comment:
can we retire this weird name "hue"? it will die in the rest of the codebase soon.

Reply:
Yes but maybe not as part of this PR